fragment.go

```diff
@@ -320,9 +310,6 @@ func (f *Fragment) closeStorage() error {
 
 	// Unmap the file.
 	if f.storageData != nil {
```

Comment:
might be clear just to remove this if statement and always set `f.storageData = nil`